### DISCUSSION TITLE
🔨 iLogtail开发者双周会（9月21号 19:30-20:30）

### DISCUSSION THREAD

**Discussion by messixukejia** [alibaba/ilogtail#388] [Announcements]
主题：iLogtail双周会
时间：9月21日 周三 19:30 - 20:30
会议号：439 549 22618
电话呼入：+862281944292（中国大陆）+867936168939（中国大陆）
入会链接：dingtalk://dingtalkclient/page/videoConfFromCalendar?confId=7fcec6bb-714f-46de-aa74-76661f98a9ac&appendCalendarId=1&calendarId=1989611942

### 议题
* 参会人自我介绍
* iLogtail社区Roadmap @messixukejia 
* 编译支持情况 @yyuuttaaoo 
* 服务端管控协议及实现进展 @Takuka0311  
* 第三方格式转换进展 @henryzhx8 
* Kafka Flusher进展 @shalousun
* Clickhouse Flusher进展 @askuy 
* OTLP输入、输出支持进展 @messixukejia @liuhaoyang 
* jmxfetch进展分享 @EvanLjp 
* 自由讨论 

### 双周会纪要
1、iLogtail社区同学相互介绍，来自同程旅行、字节、SLS、小红书、石墨（请假）的多位同学进行了自我介绍。
2、@messixukejia 介绍了社区的[Roadmap](https://github.com/alibaba/ilogtail/discussions/422)，该列表也会持续更新该列表。开发者可以就关注的功能进行留言交流，也可以提出新的特性就行交流。
3、@yyuuttaaoo 就编译的支持情况进行了介绍，[ARM](https://github.com/alibaba/ilogtail/pull/357)、[Go版本升级到1.18](https://github.com/alibaba/ilogtail/pull/375/files)、[Go vedor优化](https://github.com/alibaba/ilogtail/pull/412)、[Windows编译(期待社区共建)](https://github.com/alibaba/ilogtail/issues/327)、编译镜像在上海/美国地域归档方便就近下载。
4、@Takuka0311 介绍了[服务端管控](https://github.com/alibaba/ilogtail/blob/main/docs/cn/config-server/quick-start.md)的进展
  - [协议](https://github.com/alibaba/ilogtail/tree/main/config_server/service/proto)初步版本已制定，进一步完善中。
  - [服务端部分](https://github.com/alibaba/ilogtail/tree/main/config_server/service)目前已合入。
  - [Agent部分](https://github.com/alibaba/ilogtail/pull/406) PR中。
  - 后续也会有一个前端控制台推出。

5、为了便于开发者开发新的Flusher，@henryzhx8 开发了[协议转换接口](https://github.com/alibaba/ilogtail/tree/main/docs/cn/developer-guide/log-protocol)，开发者也可以方便的进行[协议扩展](https://github.com/alibaba/ilogtail/blob/main/docs/cn/developer-guide/log-protocol/How-to-add-new-protocol.md)。目前已在Kafka Flusher、OTLP Flusher开发中得到应用。
6、社区同学 @shalousun 就社区比较关注的kafka输出到功能进行了解答，目前已经启动Kakfa Flusher开发，近期会提PR。[相关设计](https://github.com/alibaba/ilogtail/discussions/280)，可以随时留言交流。
7、社区同学@liuhaoyang 分享了[OTLP log flusehr的进展](https://github.com/alibaba/ilogtail/pull/418/files)，目前初步代码已具备，正在进行完善及测试工作，不久将会合入主干。后续也会一起就时序等过多场景的支持展开进一步的合作。@messixukejia也分享了HTTP协议接收[OTLP Log](https://github.com/alibaba/ilogtail/pull/419)的进展，后续也会联合社区一起推出gRPC的协议支持。
8、@EvanLjp 分享了jmx fetch 的一些进展，后续版本发布后将会进一步宣传。
9、社区同学@szy441687879在使用iLogtail过程中，也提出了很多有价值的建议。例如[Go GC新特性的支持](https://github.com/szy441687879)等。也分享了一下iLogtail级联场景的一些使用经验。

[会议回放](https://www.bilibili.com/video/BV1Te4y1b7m9/?spm_id_from=333.999.0.0)，因时间冲突错过的小伙伴可以在这里观看。
最后，有兴趣参与iLogtail开发的小伙伴可以在群里@徐可甲，并留言“希望加入开发者群共建”，即可加入iLogtail的核心开发者群，在这里可以进行更充分的开发、技术交流。

**Comment by liuhaoyang** [comment]
第一次的社区会议建议 是否添加一个社区的规划和介绍、以及参会人的个人介绍环节？方便大家互相熟悉一下

**Reply by messixukejia** [reply]
很好的建议，已添加
